fuzzbench/worker.py

```diff
@@ -11,3 +11,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Self-defined worker module."""
+import redis
+import rq
+from rq import Queue, Worker
```

Comment:
we typically import modules, not classes. can we just keep the import rq above and use rq.Queue and rq.Worker in the code?

Reply:
Done.